### DISCUSSION TITLE
InputOTP unstyled is not working correctly

### DISCUSSION THREAD

**Discussion by ManoLuuL** [primefaces/community#2423] [PrimeReact] [answered]
Using the InputOTP with unstyled enabled, after updating the package to a new version, I lost my styling. This is due to typing issues because when I pass my styling with "as InputOtpPassThroughOptions", it is correctly applied to the component again. This is because the component does not recognize the path input -> root that the typing InputOtpPassThroughOptions requires.

Using according to the typing
![image](https://github.com/user-attachments/assets/94f4689c-7e41-437f-959f-15859e5311bc)

using "as InputOtpPassThroughOptions"
![image](https://github.com/user-attachments/assets/1d5eed6a-d893-4d22-b4c6-77fb8a197b02)
![image](https://github.com/user-attachments/assets/923e2ef4-345f-44e2-8745-639059527a7f)

**Comment by nitrogenous** [comment]
Could you please share a StackBlitz demo with us? This will enable us to assist you more efficiently

**Reply by ManoLuuL** [reply]
I made a simple example that represents the error.
link: https://codesandbox.io/p/devbox/splitbutton-g566mp

**Reply by melloware** [reply]
@ManoLuuL here is an easiser StackBlitz.  I see this issue.

https://stackblitz.com/edit/hwimb2?file=src%2FApp.tsx

**Reply by melloware** [reply, marked as answer]
@ManoLuuL created issue: https://github.com/primefaces/primereact/issues/6964
